postcss.config.js

```diff
@@ -1,3 +1,42 @@
+const platformPostcssConfig = require('@hashicorp/platform-postcss-config')
+
+/**
+ * TODO: this is a spike to show that this configuration may resolve
+ * the issues we're having in dev-dot when using @custom-media.
+ * We likely want to incorporate this into `@hashicorp/platform-postcss-config`.
+ * Eg, maybe it should accept an optional "options" object, where we can
+ * pass additional files to `importFrom`?
+ */
+function alsoImportDevDotCustomMedia(postcssConfig) {
+  const newPlugins = postcssConfig.plugins.map((p) => {
+    // we only want to modify postcss-preset-env
+    const isPresetEnv =
+      Array.isArray(p) && p.length == 2 && p[0] == 'postcss-preset-env'
+    if (!isPresetEnv) return p
+    // we want to modify the postcss-preset-env settings object,
```

Comment:
We could destructure this to make the following code a bit easier to grok, instead of using `p[1]`.

```ts
const [, presetEnvOptions] = plugin
```

Reply:
✅ Updated!